sample/src/main/java/vehicles/AbstractVehicle.java

```diff
@@ -0,0 +1,40 @@
+package vehicles;
+
+import people.Person;
+
+import java.time.LocalDate;
+
+/**
+ * An abstract class for implementing different kinds of {@link Vehicle}.
```

Comment:
Sorry to insist in this point, but I think this is a major issue. If such javadoc links are not converted to asciidoc ones, after the developer generates the final documentation for deployment, such elements will not be linked.

It is pretty easy do convert the link tag and it just requires very few lines of code. About the other tags such @author, @param and @return, maybe there is not need to convert. They would just be included after a "Parameters" section in order to simply organize the params, as it is already made in the default javadoc HTML.